src/test/unit/specific_feature_tests/native-edit-registration.js

```diff
@@ -0,0 +1,55 @@
+/* global
+    QUnit,
+    prepareUnitTestModule,
+    skipKeyboardShortcutTests,
+    simulateKeyCombo
+*/
+
+"use strict";
+
+module("native-edits-registering", {setup: prepareUnitTestModule});
```

Comment:
The file is `registration` and this module is `registering`. Was that on purpose?